src/components/header.css

```diff
@@ -1,20 +1,65 @@
-* {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
-  color: black;
-}
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Roboto+Slab:wght@100&display=swap');
 
-.head {
+nav {
```

Comment:
- [OPTIONAL ]  According to the project requirement, Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please make these in order to stick to the original design:
- Kindly make sure the user icon is positional to the right of your navigation bar 😜 
- What if we increase the footer input width in order the form take the whole witdh 😜 

- Your Design:

![25last](https://user-images.githubusercontent.com/79058364/175788927-f0b6d833-c496-4e3e-b7fa-5d251d272e4c.png)



- Figma design

![bookstore](https://user-images.githubusercontent.com/79058364/162426330-10521c8c-526f-4fca-9450-dc3154d41b55.png)

Reply:
done